app/controllers/application_controller.rb

```diff
@@ -55,7 +55,7 @@ def hub?
   end
 
   def current_intake
-    current_client&.intake || (Intake.find_by_id(session[:intake_id]) unless session[:intake_id].nil?)
+    current_client&.intake || (Intake.find_by_id(session[:intake_id]) unless session[:intake_id].nil?) || current_client&.current_intake
```

Comment:
We can drop `current_client&.intake` since `current_client&.current_intake` also does this check!